CMakeLists.txt

```diff
@@ -237,8 +235,8 @@ if(TOP_LEVEL_PROJECT)
         set(PACKAGE_FILE_NAME_PREFIX  "hpccsystems-clienttools-${projname}")
     endif()
     if(NOT "${CUSTOM_PACKAGE_SUFFIX}" STREQUAL "")
-	set(CPACK_PACKAGE_NAME "${CPACK_PACKAGE_NAME}-${CUSTOM_PACKAGE_SUFFIX}")
-	set(PACKAGE_FILE_NAME_PREFIX "${PACKAGE_FILE_NAME_PREFIX}-${CUSTOM_PACKAGE_SUFFIX}")
+    set(CPACK_PACKAGE_NAME "${CPACK_PACKAGE_NAME}-${CUSTOM_PACKAGE_SUFFIX}")
```

Comment:
Why this change? Looks like indentation was correct before and is now wrong?

Reply:
It was a tab. Replaced with 4 spaces, should have been 8 spaces.